parent/pom.xml

```diff
@@ -211,6 +212,13 @@
                 <version>3.0.20</version>
             </dependency>
 
+            <!-- Doclet for inclusion of code snippets -->
+            <dependency>
+                <groupId>org.apidesign.javadoc</groupId>
+                <artifactId>codesnippet-doclet</artifactId>
+                <version>${org.apidesign.javadoc.codesnippit.version}</version>
+            </dependency>
```

Comment:
True! Plugin dependencies are not managed by dependency management, perhaps I should include it as default java doc plugin management .

> Op 7 mrt. 2016 om 14:43 heeft Andreas Gudian <notifications@github.com> het volgende geschreven:
> 
> In parent/pom.xml:
> 
> > @@ -211,6 +212,13 @@
> >                  <version>3.0.20</version>
> >              </dependency>
> >  
> > +            <!-- Doclet for inclusion of code snippets -->
> > +            <dependency>
> > +                <groupId>org.apidesign.javadoc</groupId>
> > +                <artifactId>codesnippet-doclet</artifactId>
> > +                <version>${org.apidesign.javadoc.codesnippit.version}</version>
> > +            </dependency>
> This is not required here, as the version needs to be managed at the javadoc plugin dependencies.
> 
> —
> Reply to this email directly or view it on GitHub.
>